### PR TITLE
[Doc] caps in tensor-converter

### PR DESCRIPTION
**Pull request by jaeyun-jung**
update docs - flex tensor and caps in tensor-converter.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: Documentation/data-type-and-flow-control.md [hunk 1]
```diff
@@ -75,6 +75,18 @@ Dimension = (string) [1-65535]:[1-65535]:[1-65535]:[1-65535]
...
+This is useful when an element or a model requires non-determined, dynamic data shape to process the tensors. (e.g., cropping the raw data into multiple tensors)
+
+Unlike ```other/tensor``` and ```other/tensors```, flexible tensor does not contain the data structure in pad capability.
+Instead, flexible tensor has its own data structure - [GstTensorMetaInfo](https://github.com/nnstreamer/nnstreamer/blob/main/gst/nnstreamer/include/tensor_typedef.h) - in each tensor buffer, to prevent caps negotiation with fixed type of data stream.
+When processing a buffer with the capability ```other/tensors-flexible```, developer should append or parse the tensor information in buffer using various [utility functions](https://github.com/nnstreamer/nnstreamer/blob/main/gst/nnstreamer/include/nnstreamer_plugin_api.h).
+
+The buffer of ```other/tensors-flexible``` may have single memory or multiple memory chunks.
+NNStreamer element with ```other/tensors-flexible``` capability gets the number of memories in a buffer and handles each memory as a tensor.
+Note that, it also has a limit, the maximum allowed number of memory chunks in a buffer is **16**.
```
Comment (myungjoo) [inline Documentation/data-type-and-flow-control.md:89]: Please add how memory buffers contain data including the tensor types and shapes. E.g., Line 28-35.

File: gst/nnstreamer/tensor_converter/README.md [hunk 2]
```diff
@@ -26,13 +33,14 @@ From higher priority
 
 ## Sink Pads
 
-One "Always" sink pad exists. The capability of sink pad is ```video/x-raw```, ```audio/x-raw```, ```text/x-raw``` and ```application/octet-stream```.
+One "Always" sink pad exists. The capability of sink pad is ```video/x-raw```, ```audio/x-raw```, ```text/x-raw```, ```application/octet-stream``` and ```other/tensors-flexible```.
```
Comment (myungjoo) [inline gst/nnstreamer/tensor_converter/README.md:36]: A, B, C ==> A, B, and C (a comma is missing)

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_converter/README.md]: updated

File: Documentation/data-type-and-flow-control.md (hunk 1 above)
Comment (jaeyun-jung) [reply Documentation/data-type-and-flow-control.md:89]: updated mem chunks and header

**Review by myungjoo — APPROVED** [review]
LGTM

File: gst/nnstreamer/tensor_converter/README.md [hunk 3]
```diff
@@ -26,13 +33,14 @@ From higher priority
...
-One "Always" sink pad exists. The capability of sink pad is ```video/x-raw```, ```audio/x-raw```, ```text/x-raw``` and ```application/octet-stream```.
+One "Always" sink pad exists. The capability of sink pad is ```video/x-raw```, ```audio/x-raw```, ```text/x-raw```, ```application/octet-stream```, and ```other/tensors-flexible```.
 
 ## Source Pads
 
-One "Always" source pad exists. The capability of source pad is ```other/tensor```. It does not support ```other/tensors``` because each frame (or a set of frames consisting a buffer) is supposed to be represented by a **single** tensor instance.
+One "Always" source pad exists. The capability of source pad is ```other/tensor```, ```other/tensors```, and ```other/tensors-flexible```.
 
-For each outgoing frame (on the source pad), there always is a **single** instance of ```other/tensor```. Not less and not more.
+Note that, only octet-stream supports configuring multiple tensors in outgoing buffer.
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_converter/README.md:42]: Serialized buffers(flatbuf, flexbuf and protobuf) and custom converter also support multiple tensors.

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_converter/README.md:42]: update description about default caps and sub-plugin
- only octet in default caps supports ... 
- subplugin and custom converter

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**